planning/behavior_path_planner/src/behavior_path_planner_node.cpp

```diff
@@ -106,9 +106,8 @@ BehaviorPathPlannerNode::BehaviorPathPlannerNode(const rclcpp::NodeOptions & nod
   route_subscriber_ = create_subscription<LaneletRoute>(
     "~/input/route", qos_transient_local, std::bind(&BehaviorPathPlannerNode::onRoute, this, _1),
     createSubscriptionOptions(this));
-
-  avoidance_param_ptr = std::make_shared<AvoidanceParameters>(getAvoidanceParam());
   lane_change_param_ptr = std::make_shared<LaneChangeParameters>(getLaneChangeParam());
+  avoidance_param_ptr = std::make_shared<AvoidanceParameters>(getAvoidanceParam());
```

Comment:
@zulfaqar-azmi-t4 please remove these `ptr`s and declare directly when making modules. 

```
    auto lane_change_module =
      std::make_shared<LaneChangeModule>("LaneChange", *this, getLaneChangeParam());
```

Reply:
this is made into pointer so that dynamic reconfigure can be done. so if this is removed, then we can't do dynamic reconfigure.

Reply:
Ahh, I see. I thought it is a local variable because it doesn't have suffix.

Reply:
Then, would you rename them with "_" suffix as in ROS coding rule?

x `avoidance_param_ptr`
o `avoidance_param_ptr_`

Reply:
well, this is not related to this PR. You can keep as this is now. Sorry!